src/main/java/net/liquidpineapple/pang/screens/Level.java

```diff
@@ -46,9 +48,13 @@ public static Level createFromXML(String xmlFile) throws IOException {
             for (Player player : loadPlayer(doc.getElementsByTagName("player"))) {
                 output.objectList.add(player);
             }
+            TimeSystem timeKeeper = new TimeSystem(Integer.parseInt(doc.getElementsByTagName("time").item(0).getTextContent()));
```

Comment:
This has scoping and naming issues, and a readability issue.

* The [private field](https://github.com/Fastjur/SEM-Project/pull/57/files#diff-eab98ed289b8faf34e9397943c9914f4R26) (`private TimeSystem timeKeeper` at line 26) you declare is never used.
* Why not name the variable `timeSystem` to prevent confusion?
* I recommend converting the `Integer.parseInt(doc.getElementsByTagName("time").item(0).getTextContent())` to a method `loadTime()`. Similar to the `loadPlayer` and `loadBalls` methods. This line is quite long and it's hard to see what is going on. Converting this to a method that returns an integer should solve this as it can be spread over multiple lines, and will make possible future changes much easier and less merge conflict prone.